### DISCUSSION TITLE
Practica HTML - Opinión

### DISCUSSION THREAD

**Discussion by AnnethTi** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#253] [General]
Buenas noches, apenas empecé a hacer las katas de HTML y quisiera saber si lo estoy haciendo bien jeje. 

Le tengo que poner mas CSS? O así ya estaría bien para entregar(?

Espero me puedan dar su feedback, graciass:)

![pasteleria](https://user-images.githubusercontent.com/94029931/156869966-0a72abf9-1f15-43e3-b7e6-cd3b4b161531.png)
![pastelero](https://user-images.githubusercontent.com/94029931/156869971-8889f4d0-78d2-4266-a5dd-55cf48aa49be.png)

**Comment by Ingrzs** [comment]
Asi esta bien, ya que la práctica es solo maquetación, pero es desicion tuya meterle mas cosas.
Apenas el MC subio la otra práctica en el que tienes que replicar una pagina asi que ahí si tendras que meterle css.

**Reply by JuanSalvador73** [reply]
No sabía de esa otra práctica, voy a checar.

**Comment by JuanSalvador73** [comment]
El sitio está perfecto, como dicen arriba, solo tienes que entregar la maquetación, pero entre mas completo mejor. En mi caso usaré esta práctica para irla mejorando conforme avanzamos en el curso e irle agregando lo nuevo que vayamos aprendiendo.

Saludos, buen trabajo.

**Reply by RodolfoMorquecho** [reply]
Por ejemplo la funcionalidad no debe ser total de todos los botones o apartados cierto?

**Reply by ChanLuis** [reply]
Tal vez un html de inicio, luego link al "Crea tu pastel" y poco más
